src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6082,4 +6082,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_CannotUseFunctionPointerAsFixedLocal" xml:space="preserve">
     <value>The type of a local declared in a fixed statement cannot be a function pointer type.</value>
   </data>
+  <data name="ERR_UnsupportedCallingConvention" xml:space="preserve">
+    <value>The calling convention of '{0}' is not supported by C#.</value>
```

Comment:
>by C# [](start = 60, length = 5)

It doesn't look like we ever add "by C#" to error messages.

Reply:
We often stop at "not supported" or say "not supported by the language"

---
In reply to: [413328055](https://github.com/dotnet/roslyn/pull/43355#discussion_r413328055) [](ancestors = 413328055)